lib/core/src/sdk.rs

```diff
@@ -2209,11 +2209,18 @@ impl LiquidSdk {
         &self,
         req: &PrepareRefundRequest,
     ) -> SdkResult<PrepareRefundResponse> {
+        let refund_address = self
+            .validate_bitcoin_address(&req.refund_address)
+            .await
+            .map_err(|e| SdkError::Generic {
+                err: format!("Failed to validate refund_address: {e}"),
```

Comment:
Error messages aren't specific to Rust, so I'd opt to a more user-friendly error message such as this:
```suggestion
                err: format!("Failed to validate refund address: {e}"),
```